babel.config.js

```diff
@@ -14,12 +14,18 @@ module.exports = {
         },
       },
     ],
-    '@babel/react',
+    '@babel/preset-react',
     '@babel/preset-flow',
   ],
   plugins: [
     '@babel/plugin-proposal-class-properties',
     '@babel/proposal-object-rest-spread',
     '@babel/plugin-syntax-dynamic-import',
   ],
+  overrides: [
+    {
+      include: 'packages/fields/src/Controller.js',
```

Comment:
Our browserlist target means every browser supports [ES6 classes](https://caniuse.com/#feat=es6-class), can we skip compiling _every_ class?

Reply:
I'm slightly confused, do you mean you want to skip compiling for every class? That's what currently happens before this PR but that creates a problem that if a consumer uses a compiled class to extend this class for whatever reason, this could happen if a consumer compiles a class that extends this class with Babel themselves and it also currently happens because the Admin UI uses babel-preset-react-app and we could remove babel-preset-react-app from the Admin UI but there's still the problem that a consumer could compile a class that extends this class so this fix would still be necessary.

Reply:
Ah, ok, makes sense. Let's stick with this change for now, and I'll open an issue to move to pure classes (if possible) in the future 👍